src/resources/views/crud/inc/show_fields.blade.php

```diff
@@ -5,6 +5,6 @@
         $fieldsViewNamespace = $field['view_namespace'] ?? 'crud::fields';
     @endphp
 
-    @include($fieldsViewNamespace.'.'.$field['type'], ['field' => $field])
```

Comment:
I see we're doing this in `show_fields`, but then don't we also need to do it in `fields/relationship/show_fields.blade.php` ?

Reply:
Now that I think of it, we might be using it in other places too. How about we create a new method on `Fields.php` that RETURNS the fields with proper names, to be used inside a form? Then we'd have two methods to get fields:
- `$crud->fields()`
- `$crud->fieldsWithOverwrittenNames()`

We use `$crud->fields()` in PHP classes and `$crud->fieldsWithOverwrittenNames()` in blade views, since that's the only place we need it that way.

In my opinion it'd be something like:

```php

    /**
     * Get the CRUD fields for the current operation, but with
     * their names ready to be used inside inputs. For example
     * 'relationship.attribute' => 'relationship[attribute]'
     *
     * @return array
     */
    public function fieldsWithOverwrittenNames()
    {
        $fields = $this->fields();

        foreach ($fields as $key => $field) {
            $fields[$key] = $this->overwriteFieldNameFromDotNotationToArray($field);
        }

        return $fields;
    }
```

Reply:
If we do that... would this also allow us to move `overwriteFieldNameFromEntity()` from `makeSureFieldHasNecessaryAttributes()` to `fieldsWithOverwrittenNames()`?

Basically to KNOW that `makeSureFieldHasNecessaryAttributes()` doesn't overwrite anything, it just _guesses_ stuff?